lib/matplotlib/tests/test_colors.py

```diff
@@ -395,22 +395,40 @@ def test_TwoSlopeNorm_premature_scaling():
 
 
 def test_SymLogNorm():
-    """
-    Test SymLogNorm behavior
-    """
-    norm = mcolors.SymLogNorm(3, vmax=5, linscale=1.2)
+    # Test SymLogNorm behavior
+    norm = mcolors.SymLogNorm(linthresh=3, vmax=5, linscale=1.2, base=np.e)
     vals = np.array([-30, -1, 2, 6], dtype=float)
     normed_vals = norm(vals)
-    expected = [0., 0.53980074, 0.826991, 1.02758204]
+    expected = [-0.842119, 0.450236, 0.599528, 1.277676]
```

Comment:
I think these values were just plain wrong before...

- The first value (-30) is less than `vmax` (5), so should come out as less than zero
- The second value (-1) is less than 0, so should come out as less than 0.5